### ISSUE TITLE
Speech does not work with NVDA

### ISSUE THREAD

**Issue by zorkow** [mathjax/MathJax-RespEq#19]
NVDA reads out single letters.

**Comment by pkra** [comment]
Largely works now. Tested on FF, Chrome (Win7, 8, 10, Chrome even on WinXP...)

My simple live-region demo fails on Edge, so I suspect that's an Edge bug.

Not sure how much we can expect on IE.

**Comment by zorkow** [comment]
IE has still the CORS problem. So speech will be gibberish at the moment.

**Comment by zorkow** [comment]
CORS Problem resolved. Shall we close?

**Comment by pkra** [comment]
+1
